internal/core/ports/services.go

```diff
@@ -0,0 +1,18 @@
+package ports
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"github.com/Dysproz/ports-db-microservices/internal/core/domain"
+)
+
+type JSONParseService interface {
+	Load(path string)
+	Watch() <-chan domain.Entry
+}
+
+type GRPCClientService interface {
```

Comment:
GRPC is a technology specific, so it cannot stay in core. Image that the core contains business logic used by both a server and a command line application, the cli doesn't use GRPC.
Another way to think about this is a service that implements both grpc and http transport.